src/main/scala/nl.knaw.dans.easy.dd2d/DepositToDataverseMapper.scala

```diff
@@ -54,10 +55,11 @@ class DepositToDataverseMapper(narcisClassification: Elem, isoToDataverseLanguag
     addCompoundFieldMultipleValues(citationFields, AUTHOR, ddm \ "profile" \ "creatorDetails" \ "author", DcxDaiAuthor toAuthorValueObject)
     addCompoundFieldMultipleValues(citationFields, AUTHOR, ddm \ "profile" \ "creatorDetails" \ "organization", DcxDaiOrganization toAuthorValueObject)
     addCompoundFieldMultipleValues(citationFields, AUTHOR, ddm \ "profile" \ "creator", Creator toAuthorValueObject)
-    citationFields.append(contactData)
+    addCompoundFieldMultipleValues(citationFields, DATASET_CONTACT, contactData)
     addCompoundFieldMultipleValues(citationFields, DESCRIPTION, ddm \ "profile" \ "description", Description toDescriptionValueObject)
```

Comment:
The malformed json output only exists when addCompoundField() for the same MetadataBlock is called twice. The original design didn't accout for that. Another solution would have been to join both